mscore/inspector/inspector_ambitus.ui

```diff
@@ -846,8 +846,11 @@
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
+        <property name="toolTip">
+         <string>Search through the staff for lowest and highest notes, and update the ambitus' range accordingly.</string>
+        </property>
         <property name="text">
-         <string>Update Range</string>
+         <string comment="Search through the staff for lowest and highest notes, and update the ambitus' range accordingly.">Match Staff</string>
```

Comment:
How about:

“Adjust the range of the ambitus to match the notes that are currently on the staff.”

Reply:
It is shorter, but for someone that did not know what an ambitus was until today, it is not clearer.
What about changing only your _notes_ to _range of the notes_ ?
“Adjust the range of the ambitus to match the range of the notes that are currently on the staff.”

Reply:
That sounds good to me.

Reply:
Done

Reply:
That comment string is just for the translators, not visible elsewhere

Reply:
>That comment string is just for the translators, not visible elsewhere

 Yes, I thought it wasn't precise enough for translators, and after this week and the cinsiderable amount of new strings,  i witnessed enough in the translator chat to tell myself "better be clear so their job is easier" :)